components/NotificationBar.js

```diff
@@ -117,6 +160,43 @@ class NotificationBar extends React.Component {
           <div className={`NotificationLine ${status}`}>
             <h1>{title}</h1>
             <p className="description">{description}</p>
+
+            {LoggedInUser.roles[host.slug] == 'ADMIN' && (
```

Comment:
It's better to use the helpers to do this

```suggestion
            {LoggedInUser.hasRole('ADMIN', host) && (
```